examples/train_rnnt.py

```diff
@@ -0,0 +1,130 @@
+from datasets.librispeech import iterate
+from tinygrad.helpers import LazyNumpyArray
+from tinygrad.lazy import LazyBuffer
+from tinygrad.tensor import Tensor, Function
+from tinygrad.nn.optim import LAMB, get_parameters
+import numpy as np
+
+from models.rnnt import RNNT
+
+def rnnt_loss_forward(x: Tensor, y: Tensor, blank=28):
```

Comment:
So I wrote these to take numpy arrays for a reason, because rnn-t loss has a lot of gather scatter operations which tinygrad doesn't support well, so I implemented the loss as a custom function (See: https://github.com/tinygrad/tinygrad/blob/master/test/test_custom_function.py). For speed it will probably have to be implemented as a opencl kernel, but it may also be possible to just implement it using tinygrad ops by finding workarounds for the gather scatter ops.

Reply:
Thanks @wozeparrot for the advice. I think implementing them using tinygrad ops is a better way to proceed, as kernel dependency may change drastically.